src/templates/index.html

```diff
@@ -42,7 +42,17 @@
     </div>
     <footer class="footer">
       <div class="container-fluid">
-        <p class="text-muted" style="font-size:12px;"><b>Disclaimer:</b>&nbsp;The package information is provided AS-IS with no representations whatsoever with respect to accuracy and/or completeness. We reserve the right to change the information at any time and without notice. To obtain the most current package information, please visit the official linux distribution websites.</p>
+        <p class="text-muted" style="font-size:12px;">
+          <b>Disclaimer:</b>
+          &nbsp;The package information is provided AS-IS with no representations whatsoever with respect to accuracy and/or completeness. We reserve the right to change the information at any time and without notice. To obtain the most current package information, please visit the official linux distribution websites.
```

Comment:
Capitalize the proper noun "Linux" and add at the end "or websites for specific software you're looking into."